tests/e2e/cucumber/steps/ui/resources.ts

```diff
@@ -60,22 +60,25 @@ When(
 )
 
 When(
-  /^"([^"]*)" (copies|moves) the following (resource|resources)$/,
+  /^"([^"]*)" (copies|moves) the following resource(?:s)? using the ([^"]*) ?(?:panel)?$/,
   async function (
     this: World,
     stepUser: string,
     actionType: string,
-    _: string,
+    method: string,
     stepTable: DataTable
   ): Promise<void> {
     const { page } = this.actorsEnvironment.getActor({ key: stepUser })
     const resourceObject = new objects.applicationFiles.Resource({ page })
 
     for (const { resource, to } of stepTable.hashes()) {
-      await resourceObject[actionType === 'copies' ? 'copy' : 'move']({
-        resource,
-        newLocation: to
-      })
+      await resourceObject[actionType === 'copies' ? 'copy' : 'move'](
+        {
+          resource,
+          newLocation: to
+        },
+        method
+      )
```

Comment:
move methods to object arg as well
```suggestion
      await resourceObject[actionType === 'copies' ? 'copy' : 'move'](
        {
          resource,
          newLocation: to,
          method
        }
      )
```